changelogs/unreleased/fix-incompatiblity-pytest-postgresql.yml

```diff
@@ -0,0 +1,6 @@
+---
+description: "Fix incompatibilty with pytest-postgresql when using pytest8. Due to a bugfix in pytest8, the postgresql_proc_with_log fixture now returns None instead of an empty string."
```

Comment:
```suggestion
description: "Fix incompatibility with pytest-postgresql when using pytest8. Due to a bugfix in pytest8, the postgresql_proc_with_log fixture now returns None instead of an empty string."
```